rust/migration/src/m20230707_144317_record_uses.rs

```diff
@@ -0,0 +1,53 @@
+use sea_orm_migration::prelude::*;
+
+use crate::m20220101_000001_create_table::Job;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(JobUses::Table)
+                    .if_not_exists()
```

Comment:
I may have missed this before but you shouldn't use `.if_not_exists()` in a migration if that table wouldn't have been created already by a previous migration. If this table already exists then we should fail hard because the migrator is the only one with authority to create this table and we can look at the migration history and assert it hasn't been created in the chain

Reply:
(this is another one of those things that won't ever actually come up but its another safety rail)

Reply:
Yeah that makes sense. I can go back and modify them to fit that in a follow up. For some reason it was the sea-orm way to do things

Reply:
SGTM